src/expectEvent.js

```diff
@@ -161,7 +177,10 @@ expectEvent.inLogs = deprecate(inLogs, 'expectEvent.inLogs() is deprecated. Use
 expectEvent.inConstruction = inConstruction;
 expectEvent.inTransaction = inTransaction;
 
-expectEvent.not = {};
-expectEvent.not.inConstruction = notInConstruction;
-expectEvent.not.inTransaction = notInTransaction;
+expectEvent.notEmitted = notExpectEvent;
+expectEvent.notEmitted.inConstruction = notInConstruction;
+expectEvent.notEmitted.inTransaction = notInTransaction;
+
+expectEvent.not = deprecate(inLogs, 'expectEvent.not is deprecated. Use expectEvent.notEmitted instead.');
```

Comment:
We should keep the same API as before for backwards compatibility, with the extra deprecation notices.

```suggestion
expectEvent.not = {};
expectEvent.not.inConstruction = deprecate(notInConstruction, 'expectEvent.not is deprecated. Use expectEvent.notEmitted instead.');
expectEvent.not.inTransaction = deprecate(notInTransaction, 'expectEvent.not is deprecated. Use expectEvent.notEmitted instead.');
```

Reply:
Oh damn, sorry, this was lousy work. Thanks for spotting this!